CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make by creating an [issue](https://github.com/UBC-LA-Hackathon/student-dashboard/issues).
+
+Please note we have a [code of conduct](./CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Create an issue.
+1. Fork the repo.
+1. Make your change.
+1. Create your pull request.
+1. All pull requests to `master` must be reviewed by at least 1 maintainer in order to be merged.
+1. Once the pull request is approved, it will be merged into `master`.
+   
+### Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
+available at [http://contributor-covenant.org/version/1/4][version]
```

Comment:
I clicked through this link and the Contributor Covenant seems to have more to do with the code of conduct than the the "first create issues to discuss changes, then create pull requests to merge changes" workflow. I'd suggest deleting the attribution to the contributor-covenant, or attaching that attribution to the [./CODE_OF_CONDUCT.md](./CODE_OF_CONDUCT.md). What do you think?

Reply:
I don't mind deleting the `Attribution` section entirely.